google-cloud-clients/google-cloud-dns/src/test/java/com/google/cloud/dns/it/ITDnsTest.java

```diff
@@ -2066,4 +2066,34 @@ public void testBatchCombined() {
       DNS.delete(ZONE1.getName());
     }
   }
+
+  @Test
+  public void testCaaRecord() {
```

Comment:
Do you think `testCAARecord` sounds better?

Reply:
Done

Reply:
PTAL